### PR TITLE
Add How-To develop local section to README

### PR DESCRIPTION
**Pull request by MarauderXtreme**
This helps to setup a development environment locally via docker and
use it with ease. This deepends on #15 since those plugins are needed
for this website to function without further changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,29 @@ Witzige, originelle Inhalte, die nicht weitläufig verbreitet sind.
...
+### Weiterentwicklung der visuellen Aufbereitung
+
+Diese visuelle Aufbereitung für das WWW wird mithilfe des kleinen Programms `jekyll` erstellt.
+Um die Schablonen weiter zu bringen, ist eine lokale Installation des Programms `jekyll` inklusive der Abhängigkeiten notwendig.
+
+#### Docker
+
+Um sich nicht eine komplette Umgebung mit Rubin, Edelsteinen und Bündel aufsetzen zu müssen, kann man sich Hilfe bei `Docker` holen.
+Das Abbild `jekyll/jekyll:VERSIONSNUMMER` funktioniert wunderbar, um eine lokal begrenzte Umgebung zu besitzen.
+Für Github Pages Umgebungen ist `jekyll` in der Version 3.9 gesetzt, weswegen man `jekyll/jekyll:3` benutzen sollte.
```
Comment (Lodifice) [inline README.md:28]: Bitte Eigennamen nicht ins Englische Übersetzen und Deppenleerzeichen vermeiden! Vorschlag: Deppendrehkreuzseitenumgebungen.

**Review by Lodifice — CHANGES_REQUESTED** [review]
Apart from my comment, this looks fine to me.

File: README.md [hunk 2]
```diff
@@ -15,3 +15,29 @@ Witzige, originelle Inhalte, die nicht weitläufig verbreitet sind.
...
+
+Um die Schreibarbeit zu vereinfachen, sind folgende handliche Befehle als Aliase nützlich.
+Ein Programm wie zum Beispiel [`direnv`](https://direnv.net/) kann hier bei multiplen Versionen helfen.
+Zusätzlich wird eine lokale Version der gebündelten Edelsteine abgelegt und agiert damit als Schnellzugriffszwischenspeicher.
+
+```sh
+export JEKYLL_VERSION=3
+alias jekyll='docker run -it --rm -v "$PWD":/srv/jekyll -v "$PWD/vendor/bundle":/usr/local/bundle -p 127.0.0.1:4000:4000 jekyll/jekyll:"$JEKYLL_VERSION" jekyll'
```
Comment (Lodifice) [inline README.md:36]: $PWD ist keine Standardmuschelvariable, bitte verwenden Sie einen Untermuschelaufruf mit dem Eingebauten pwd.

File: README.md (hunk 1 above)
Comment (MarauderXtreme) [reply README.md:28]: You are sending conflicting signals here.
Either I should not translate or I should.

Comment (Lodifice) [reply README.md:28]: Oh yes, there it is! I finally retained sanity and was able to comprehend the ingeniousness of my statement! 

There is no contradiction. I asked you to not translate proper names like Deppendrehkreuz into English.

Comment (MarauderXtreme) [reply README.md]: There you go buddy.
c2f67b431e3feea8908379a2e7a3d78719affa37
This one is on you, answering to a resolved thread.

**Review by Lodifice — APPROVED** [review]
Thanks a lot! Now we finally have clear and understandable instructions for everyone.

**Pull request merged**